Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/ReflectometryISISCalibration.py

```diff
@@ -16,14 +14,18 @@
 )
 from mantid.kernel import Direction
 import csv
+import collections
 import math
 
 
 class ReflectometryISISCalibration(DataProcessorAlgorithm):
     _WORKSPACE = "InputWorkspace"
     _CALIBRATION_FILE = "CalibrationFile"
     _OUTPUT_WORKSPACE = "OutputWorkspace"
-    _DEBUG = "Debug"
+
+    _DET_ID_LABEL = "detectorid"
```

Comment:
Is there a reason this isn't `detector_id`? Just to make it the same as the theta label